cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java

```diff
@@ -465,7 +466,7 @@ private void createSession( File target, File settings )
 
             if ( ar != null)
             {
-                ar.setUrl( "file://" + req.getLocalRepositoryPath() );
+                ar.setUrl( req.getLocalRepositoryPath().toPath().toUri().toString() );
```

Comment:
I assume that `File.toURI()` is the same as `Path.toUri()`, but just to be safe I converted it to a `Path` first. There is something int he javadoc about `File.toURL()` not being safe, in any case.

Reply:
As per above, I think this needs rebasing.